docs/plugins_es.md

```diff
@@ -0,0 +1,373 @@
+  /** Permitir en cualquier otra configuración posible, depende de los plugins */
+  [key: string]: any;
+}
+
+export interface HandledRoute {
+  /** La ruta _completa_ */
+  route: string;
+  /** String, debe ser el nombre de un plugin existente */
+  type: string;
+  /** la parte relevante de scully-config  */
+  config?: RouteConfig;
+  /** variables expuestas a angular _¡solo mientras se hace el render!_ */
+  exposeToPage?: {
+    manualIdle?: boolean;
+    transferState?: Serializable;
+    [key: string]: Serializable;
+  };
+  /** datos que serán inyectados en la página estática */
+  injectToPage?: {
+    [key: string]: Serializable;
+  };
+  /** arreglo con los nombres de los plugins que serán ejecutados */
+  postRenderers?: string[];
+  /** el "path" del archivo de contenido */
+  templateFile?: string;
+  /**
+   * datos adicionales que terminarán en el archivo scully.routes.json
+   * los datos del frontMatter también serán agregados aquí.
+   */
+  data?: RouteData;
+}
+```
+
+La interface `HandledRoute` proporciona las propiedades necesarias para desarrollar su propio plugin.
+
+### route: string
+
+`route` - Una ruta de aplicación que será manejada por Scully. Esta es la información _completamente calificada_ de la route. Esto quiere decir que no debe haber variables restantes aquí. Además el símbolo `#` no está permitido y los parámetros query de ruta son ignorados.
+
+### type: RoutesTypes
+
+`type` - Indica el tipo de plugin. Contiene el nombre del plugin de ruta. Este campo es obligatorio y _debe_ ser proporcionado. Cuando el tipo no existe, Scully finaliza ya que no sabe que hacer.
+
+### defaultPostRenderers?: string[]
+
+`defaultPostRenderers` - Arreglo con ID's de tipo string con los content-renderers que serán ejecutados en todas las rutas.
+
+### postRenderers?: string[]
+
+`postRenderers` - Arreglo de nombres de plugins que se ejecutarán desplues de mostrar la página de inicio. Cada uno de los plugins en este arreglo se mostrarán en el orden en el que aparecen y recibiran el HTML resultante del plugin anterior. 
+Además, teste arreglo _reemplaza_ el arreglo `defaultPostRenderers`. 
+
+```typescript
+const defaultPostRenderers = ['seoHrefOptimise'];
+const sampleConf: ScullyConfig = {
+  defaultPostRenderers,
+  routes: {
+     /** obtiene los postrenderes por defecto */
+    normalRoute: {
+      type: 'default'
+    },
+    /** agrega los postrenderes por defecto */
+    someRoute: {
+      type: 'default',
+      postRenderers: [...defaultPostRenderers, 'myAddition']
+    },
+    /** remueve los postrenderes por de fecto */
+    someOtherRoute: {
+      type: 'default',
+      postRenderers: ['unique']
+    }
+  }
+};
+````
+
+Los `defaultPostRenderers` y los `postRenderers` están diseñados  de esta manera con la finalidad de permitirle deshacerse de los renderers por defecto. 
+Además, este diseño actual de los renders es versátil, flexible y opt-out de manera facil.
```

Comment:
```markdown
Además, este diseño actual de los renders es versátil, flexible y opt-out de manera facil.
```

Hay 2 cosas acá que acotar: 
1) La frase, "flexible y opt-out", que quiere decir esto?
2) Falta el acento en "fácil"

Reply:
Flexible: Que se adapta fácilmente a los cambios y a las diversas situaciones o circunstancias.
opt-out: Is a common terminology in CS and engineering in general. I'll try to explain this term in Spanish